lib/semian/net_http.rb

```diff
@@ -0,0 +1,85 @@
+require 'semian'
+require 'semian/adapter'
+require 'net/https'
+require 'open-uri'
```

Comment:
But why would you have theses? `open-uri` is nice for quick and dirty script, but it's the last thing you want to be required in your application.